### PR TITLE
MAINTAIN: add jobmon dependency with version 0.0.5

### PR DESCRIPTION
**Pull request by zhengp0**

### CODE REVIEW

**Review by mbannick — APPROVED** [review]
Thanks, @zhengp0 !

File: setup.py [hunk 1]
```diff
@@ -16,15 +16,15 @@
         "intervaltree",
         "pytest",
         "tables",
-        "networkx"
+        "networkx",
+        "jobmon==0.0.5"
```
Comment (mbannick) [inline setup.py]: this has to go in the `ihme` section below because travis isn't able to find it

**Review by mbannick — CHANGES_REQUESTED** [review]

File: setup.py [hunk 2]
```diff
@@ -26,7 +25,9 @@
             "db-queries",
             "gbd",
             "elmo",
-            "ihme-rules"],
+            "ihme-rules",
+            "jobmon==0.0.5"
```
Comment (zhengp0) [inline setup.py:29]: Thanks @mbannick ! Let me know if this is good :)

Comment (mbannick) [reply setup.py:29]: yes!! looks great!

**Review by mbannick — APPROVED** [review]

**Pull request merged**